types/wojtekmaj__react-datetimerange-picker/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @wojtekmaj/react-datetimerange-picker 3.0
+// Project: https://github.com/wojtekmaj/react-datetimerange-picker (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: Pesven <https://github.com/Psvensso>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
+
+import { CalendarProps } from 'react-calendar';
+
+export default function DateTimeRangePicker(props: DateRangePickerProps): JSX.Element;
+
+export interface DateRangePickerProps extends CalendarProps {
+    amPmAriaLabel?: string;
+    autoFocus?: boolean;
+    calendarAriaLabel?: string;
+    calendarClassName?: string;
+    calendarIcon?: string | JSX.Element;
+    clearAriaLabel?: string;
+    clearIcon?: string | JSX.Element;
+    clockClassNamestring?: string;
+    closeWidgets?: boolean;
+    dayAriaLabelstring?: string;
+    disabled?: boolean;
+    disableCalendar?: boolean;
+    disableClock?: boolean;
+    format?: string;
+    hourAriaLabel?: string;
+    isCalendarOpen?: boolean;
+    isClockOpen?: boolean;
+    minuteAriaLabel?: string;
+    monthAriaLabel?: string;
+    name?: string;
+    nativeInputAriaLabel?: string;
+    onCalendarClose?: () => void;
+    onCalendarOpen?: () => void;
+    onClockClose?: () => void;
+    onClockOpen?: () => void;
+    rangeDivider?: string;
```

Comment:
onChange, onFocus?

Reply:
I did not go through the proptypes thouroly enough i see, you should also update the documentation on the readme. (onFocus and a few others are missing there). 
Added a few missing **Placeholder string and the rest that were previously taken from calendar (like value) so now it's 1-1 with the [proptypes](https://github.com/wojtekmaj/react-datetimerange-picker/blob/b9941b90c11f81bc4a31a8d26bc2c2b70bf6c5b9/src/DateTimeRangePicker.jsx#L490)